### PR TITLE
itsarraj

### PR DESCRIPTION
**Pull request by itsarraj**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6509"></a>

### CODE REVIEW

**Comment by itsarraj** [comment]
@eddiejaoude @pradhuman09

**Review by Arindam200 — DISMISSED** [review]

**Review by itsarraj — COMMENTED** [review]
Good

**Review by accodes21 — DISMISSED** [review]

File: data/itsarraj.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "tags": [
+    "Open Source",
+    ,"C","C++","Python","Java","Javascript","Go","NodeJS","ReactJS","MongoDB","MYSQL","Linux"
+  ],
+  "links": [
+    {
+      "name": "Github",
+      "url": "https://github.com/itsarraj",
+      "icon": "FaGithub"
+    },
```
Comment (SaraJaoude) [inline data/itsarraj.json:18]: delete the comma:
```suggestion
    }
```

File: data/itsarraj.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "name": "Animesh Raj",
+  "type": "personal",
+  "bio": "Full Stack Developer & Software Engineer🔥Open Source🚀📈 Committing every day!",
+   "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/itsarraj" },
+    { "icon": "FaGithub", "url": "https://github.com/itsarraj" },
+  ],
+  "tags": [
+    "Open Source",
+    ,"C","C++","Python","Java","Javascript","Go","NodeJS","ReactJS","MongoDB","MYSQL","Linux"
```
Comment (SaraJaoude) [inline data/itsarraj.json:11]: delete comma:
```suggestion
    "C","C++","Python","Java","Javascript","Go","NodeJS","ReactJS","MongoDB","MYSQL","Linux"
```

**Comment by SaraJaoude** [comment]
> @ eddiejaoude @ pradhuman09


@itsarraj  please do not tag individual maintainers - any maintainer can review your Pull Request and tagging creates unnecessary notifications. Please familiarise yourself with the Contributing Guide which explains how we run this project:https://github.com/EddieHubCommunity/LinkFree/blob/main/CONTRIBUTING.md

You would also have been notified that the `JSON check` did not pass - this would have given you the opportunity to correct the JSON which would have meant your PR would have been merged faster.

**Comment by SaraJaoude** [comment]
@Arindam200 and @accodes21 please make sure the actions have passed before approving - or else this gives more confusion to the author and maintainers.

**Comment by Arindam200** [comment]
> @Arindam200 and @accodes21 please make sure the actions have passed before approving - or else this gives more confusion to the author and maintainers.

I'm extremely Sorry for that! I'll keep this in mind!

**Comment by eddiejaoude** [comment]
@itsarraj did you mean to close your pull request?

**Comment by itsarraj** [comment]
yes , have to reopen , it was not passing tests , have to setup again

**Comment by eddiejaoude** [comment]
ah ok, you can add to an existing pull request, but if easier, yes you can also open a new pull request 👍

**Comment by itsarraj** [comment]
@fabian4 @itsarraj @SaraJaoude @eddiejaoude @eddiejaoude This guy is scammer scam people in name of open source contributions guys be aware of these scams

**Comment by itsarraj** [comment]
@eddiejaoude this guy is motherfucker

**Comment by itsarraj** [comment]
@eddiejaoude This guy is scammer scam people in name of open source contributions guys be aware of these scams

**Comment by itsarraj** [comment]
@eddiejaoude This guy is scammer scam people in name of open source contributions guys be aware of these scams

**Comment by itsarraj** [comment]
@kunal-kushwaha this guy is another scammer

**Comment by itsarraj** [comment]
@Arindam200 @accodes21 @SaraJaoude @daba-favalcodes @GaganpreetKaurKalsi @eddiejaoude This guy is scammer scam people in name of open source contributions guys be aware of these scams

**Comment by max-programming** [comment]
Came here from your tweet. Happy Birthday Eddie!

**Pull request closed without merging**